app/src/main/java/me/guillaumin/android/osmtracker/activity/TrackDetail.java

```diff
@@ -263,6 +294,15 @@ public void onItemClick(AdapterView<?> parent, View view, final int position, fi
 	}
 	
 	/**
+	 * On track path changed, update track info
+	 */
+	private void pathChanged() {
+		updateTrack();
+		// Update the list view
+		lv.setAdapter(lv.getAdapter());
```

Comment:
That's a hack indeed. There must be a way to trigger a refresh of the list view... (but I don't know off hand sorry)